src/vehicle.cpp

```diff
@@ -354,6 +354,14 @@ void vehicle::init_state( int init_veh_fuel, int init_veh_status )
         }
     }
 
+    // Fuel in some vehicles will be syphoned by other survivals.
+    int fuel_reducing = get_option< int >( "VEHICLE_FUEL_REDUCING" );
+    bool has_fuel = 1;
```

Comment:
Why are you initializing a `bool` with an `int`?

Reply:
Isn't it a standart covnersion? This variable will be set to 1 or 0 later, so you suggest to change its type to `int`?

Reply:
boolean variables have value true or false.